sematic/db/migrations/20230615104836_add_organizations_table.sql

```diff
@@ -0,0 +1,36 @@
+-- migrate:up
+CREATE TABLE IF NOT EXISTS organizations (
+    id character(32) NOT NULL,
+    name TEXT NOT NULL,
+    namespace TEXT,
+    created_at TIMESTAMP WITH time zone NOT NULL,
+    updated_at TIMESTAMP WITH time zone NOT NULL,
+
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS organizations_users (
+    organization_id character(32) NOT NULL,
+    user_id character(32) NOT NULL,
+    admin BOOLEAN NOT NULL,
+    created_at TIMESTAMP WITH time zone NOT NULL,
+    updated_at TIMESTAMP WITH time zone NOT NULL,
+
+    PRIMARY KEY (organization_id, user_id),
+
+    FOREIGN KEY(organization_id) REFERENCES organizations (id),
+    FOREIGN KEY(user_id) REFERENCES users (id)
+);
+
+INSERT INTO organizations (id, name, namespace, created_at, updated_at)
+SELECT id, email, NULL, created_at, created_at
```

Comment:
So personal orgs names are email addresses? Anything smarter we can do? I think we are guaranteed to have user's first name since they sign up with Google. So we could name them "First Name Last Name".